src/main/java/slimeknights/tconstruct/tables/network/ToolStationSelectionPacket.java

```diff
@@ -0,0 +1,81 @@
+package slimeknights.tconstruct.tables.network;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.entity.player.ServerPlayerEntity;
+import net.minecraft.inventory.container.Container;
+import net.minecraft.item.ItemStack;
+import net.minecraft.network.PacketBuffer;
+import net.minecraft.world.server.ServerWorld;
+import net.minecraftforge.fml.LogicalSide;
+import net.minecraftforge.fml.network.NetworkEvent;
+import slimeknights.mantle.inventory.BaseContainer;
+import slimeknights.mantle.network.AbstractPacket;
+import slimeknights.tconstruct.library.network.TinkerNetwork;
+import slimeknights.tconstruct.tables.client.inventory.table.ToolStationScreen;
+import slimeknights.tconstruct.tables.inventory.table.toolstation.ToolStationContainer;
+
+import java.util.function.Supplier;
+
+public class ToolStationSelectionPacket extends AbstractPacket {
+
+  public ItemStack tool;
```

Comment:
Packet parameters should be `private final`, helps find errors like forgetting to set one.